15-draft.json

```diff
@@ -816,6 +816,53 @@
             ]
           }
         },
+        "power_generation": {
+          "description": "The power generation of a specific device or of your whole space",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "value": {
+                "description": "The sensor value",
+                "type": "number"
+              },
+              "unit": {
+                "description": "The unit of the sensor value. You should always define the unit though if the sensor is a flag of a boolean type then you can of course omit it.",
```

Comment:
The description does seem a bit off? How can the sensor be of boolean type? Also since it is a required field one can't omit it.

Reply:
Actually, I copy&pasted from `power consumption` and did not have a close enough look at it.
So anything that you might request here is probably also wrong at `power_consumption` :)

Reply:
I have remove the misleading part of the description in [961e696](https://github.com/SpaceApi/schema/pull/105/commits/961e6968343fe723f4a79367c543b4b381989d09).

Reply:
Thanks we'll fix this for the other sensor.